ibm_cloud_sdk_core/utils.py

```diff
@@ -88,6 +89,32 @@ def string_to_datetime(string: str) -> datetime.datetime:
     """
     return date_parser.parse(string)
 
+def convert_model(val):
```

Comment:
I'd like to add type-hints on this method but not sure how to do that.  The input `val` could be anything.  The output will be a `dict` if the input had a `to_dict` method, but otherwise could be anything.  So is the right answer:
```
def convert_model(val: Any) -> Any:
```
and if yes, then is it even worth providing type hints?